logger/logger.go

```diff
@@ -46,16 +47,22 @@ func SetLogger(zl *zap.Logger) {
 func CreateDiskLogger(dir string, lvl zapcore.Level) *zap.Logger {
 	config := zap.NewProductionConfig()
 	destination := path.Join(dir, "log.jsonl")
-	config.OutputPaths = []string{"stderr", destination}
-	config.ErrorOutputPaths = []string{"stderr", destination}
+	config.OutputPaths = []string{"pretty", destination}
+	config.ErrorOutputPaths = []string{"pretty", destination}
 	config.Level.SetLevel(lvl)
-	zl, err := config.Build(zap.AddCallerSkip(1))
+	zl, err := config.BuildWithSinks(generateSinks(), zap.AddCallerSkip(1))
 	if err != nil {
 		log.Fatal(err)
 	}
 	return zl
 }
 
+func generateSinks() map[string]zap.Sink {
+	factories := zap.DefaultSinks()
+	factories["pretty"] = PrettyConsole{os.Stdout}
```

Comment:
How do you feel about continuing to use `stderr`?

Reply:
Most of these statements are `INFO` or `DEBUG` which leads me to believe it's stdout rather than stderr. Also, it is the general output of the application. Why would you prefer stderr?